homu/main.py

```diff
@@ -1064,6 +1064,8 @@ def arguments():
                     'continuous integration service')
     parser.add_argument('-v', '--verbose',
                         action='store_true', help='Enable more verbose logging')
+    parser.add_argument('-c', '--config',
+                        action='store', help='Path to cfg.toml', default='cfg.toml')
```

Comment:
Should this have no default maybe? To differentiate in the fallback below between not providing the option and providing `--config cfg.toml`.